plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -495,6 +501,14 @@ class MarathonStore extends GetSetBaseStore {
         };
       }
 
+      if (item instanceof Application) {
+        map[item.getName().toLowerCase()] = {
```

Comment:
This will also affect line 517 https://github.com/dcos/dcos-ui/pull/3820/files#diff-7b6a93d2ba53835d3ed69d3f2e183f45R531

We used to collect only frameworks there now all the apps will also land into the resulting structure with unknown consequences.

Reply:
The way I understand it, our framework check is not ideal, since we only check a label. So everything with the label is a framework.

Reply:
exactly right, we consider everything that has `DCOS_PACKAGE_FRAMEWORK_NAME` to be a framework, though for icons we check for another label, namely `DCOS_PACKAGE_NAME` not every catalog package is a Mesos Framework

Reply:
This is why I want to allow Applications that have images to show them.

Reply:
Are we sure that there will be no side effects augmenting this data structure?

Reply:
I am not absolutely sure, but I can't think of any. If you do, please share so I can handle these.